src/integritysetup/integrity-util.c

```diff
@@ -34,9 +34,18 @@ int parse_integrity_options(
                 else if (streq(word, "allow-discards")) {
                         if (ret_activate_flags)
                                 *ret_activate_flags |= CRYPT_ACTIVATE_ALLOW_DISCARDS;
-                } else if (streq(word, "no-journal")) {
-                        if (ret_activate_flags)
-                                *ret_activate_flags |= CRYPT_ACTIVATE_NO_JOURNAL;
+                } else if ((val = startswith(word, "mode="))) {
+                        if (streq(val, "journal")) {
+                                /* Default mode, no action */
```

Comment:
so here, please turn off both bits

Reply:
Will fix. Do you prefer to have the fixup as a fixup commit or a force push to the branch?

Reply:
Nvm, you already answered that. Will force-push.